.github/workflows/ci.yml

```
@@ -61,17 +61,26 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+  
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+  
       - name: Install tox
         run: |
           python -m pip install --upgrade pip
           pip install tox
+
       - name: Run tox
         run: tox -e ${{ matrix.toxenv }}
 
+      - name: Upload coverage to codecov
+        if: ${{ contains(matrix.toxenv,'-cov') }}
```

Your tox env that does coverage is named "coverage", so you may want to change the following 
```suggestion
        if: ${{ contains(matrix.toxenv,'coverage') }}
```
On the `jwst` tox envs, we do env names like `py39-cov` to do coverage, so we search for "-cov" to trigger the upload action